vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/HttpCookieSteps.java

```diff
@@ -89,4 +92,40 @@ public void saveHttpCookieIntoVariable(String cookieName, Set<VariableScope> sco
             }
         }
     }
+
+    /**
+     * Change cookie value.
+     * If present several cookies with the same name will be changed value for all of them,
```

Comment:
`If several cookies with the same name exist in cookie store, the value will be changed for all of them`